app/api/csv/specs/csvExporter.spec.ts

```diff
@@ -0,0 +1,576 @@
+    name: 'Comic character',
+    properties: [
+      {
+        _id: '58ad7d240d44252fee4e6200',
+        id: 'f6c30c8a-3013-471f-be8c-e639f6aeb031',
+        name: 'company',
+        type: 'text',
+        label: 'company',
+      },
+      {
+        _id: '58ad7d240d44252fee4e61ff',
+        relationType: '5aae90e0bfbf88e5ae28b19e',
+        id: '3780239a-f858-4df5-9dea-8ea4a59cfc9e',
+        name: 'nemesis',
+        content: '58ad7d240d44252fee4e61fb',
+        type: 'relationship',
+        label: 'Nemesis',
+      },
+      {
+        _id: '5e3d19ccdeeb2652690a1258',
+        label: 'Costume',
+        type: 'select',
+        content: '5e3d1853deeb2652690a0c10',
+        showInCard: false,
+        filter: false,
+        name: 'costume',
+        id: 'a5407e10-5148-4241-ad81-0c8fa78f1c43',
+      },
+      {
+        _id: '58ad7d240d44252fee4e61fe',
+        filter: true,
+        id: '113d9a13-7fb9-447f-abf1-4075a9f8eb00',
+        name: 'super_powers',
+        content: '58ad7d240d44252fee4e6208',
+        type: 'multiselect',
+        label: 'Super powers',
+      },
+      {
+        _id: '59859ad8ddb12b0ce6664927',
+        relationType: '5a8480fac464318833d9b553',
+        label: 'Allies',
+        type: 'relationship',
+        content: '58ad7d240d44252fee4e61fb',
+        name: 'allies',
+        id: 'c22326ac-6723-42b5-bb3e-de0fdcc2e2dc',
+      },
+      {
+        nestedProperties: [],
+        _id: '5e8f9509f16db8b791fec574',
+        label: 'Geolocation',
+        type: 'geolocation',
+        name: 'geolocation_geolocation',
+        id: 'e9b810a9-8f25-442f-b521-616f3f3bbcdd',
+      },
+    ],
+  },
+  '58ad7d240d44252fee4e61fb': {
+    _id: '58ad7d240d44252fee4e61fb',
+    name: 'Super Villian',
+    properties: [
+      {
+        _id: '5e3d1880deeb2652690a1036',
+        label: 'Costume',
+        type: 'select',
+        content: '5e3d1853deeb2652690a0c10',
+        name: 'costume',
+        id: '53d6bb4a-2819-47b5-95a3-9261da5e8a69',
+      },
+      {
+        _id: '58ad7d240d44252fee4e61fc',
+        id: '58dd46d2-b52c-4e80-a859-6f4fadabe4c0',
+        name: 'super_powers',
+        content: '58ad7d240d44252fee4e6208',
+        type: 'multiselect',
+        label: 'Super powers',
+      },
+      {
+        _id: '594bc3b0bee8b3829aea937f',
+        relationType: '5aae90e0bfbf88e5ae28b1a3',
+        label: 'Sidekick',
+        type: 'relationship',
+        content: '58f0aed2e147e720856a0741',
+        name: 'sidekick',
+        id: '0c45e6dc-0081-463e-9300-c46d13b1dcd2',
+      },
+      {
+        _id: '594bc3b0bee8b3829aea937e',
+        label: 'Planets conquered',
+        type: 'numeric',
+        name: 'planets_conquered',
+        id: '6ec2c27a-ec30-4d0f-a7da-217a57e40ef2',
+      },
+      {
+        _id: '594bc3b0bee8b3829aea937d',
+        label: 'DOB',
+        type: 'date',
+        name: 'dob',
+        id: '87289f51-21fb-4ff6-bb8d-8b7d66d86526',
+      },
+    ],
+  },
+};
```

Comment:
Perhaps this fixtures are long enough to merit a 'fixtrues' file like in many other api tests?